README.md

```diff
@@ -124,6 +124,39 @@ If you want to check what resources are going to be targeted without actually te
 cloud-nuke aws --resource-type ec2 --dry-run
 ```
 
+### Specify optional config file
+
+For more granularity, you can pass in a config file to specify which resources to terminate using regular expressions.
+
+```shell
+cloud-nuke --config path/to/file.yaml
+```
+
+For example, if you want to terminate S3 buckets whose name matches the given patterns, here's what your config file might look like:
+```yaml
+s3:
+  name_filter:
+    - ^alb-alb-.*-access-logs$
+    - .*-prod-alb-.*
+```
+
+If you want to terminate S3 buckets by region:
+```yaml
+s3:
+  region_filter:
+    - us-east-1
+    - us-west-1
+```
```

Comment:
Same as above. Having one extra sentence to clarify what this exactly means ("nuke s3 buckets in the regions `us-east-1` and `us-west-1`") would help here.

Reply:
Also, how does this work if you provide `--exclude-region` or `--region` in the CLI? Does it override? What if the CLI arg is a region that is not in the config (e.g. `--region eu-west-1` with this config)?

Reply:
I propose that the options provided in as CLI options should take precedence over those provided in the config file. This is simlar to how passing in environment variables by prepending their assignment to the CLI command would override those variables in the CLI function.

I'll add this note in the README to clarify this! I think we still might want to discuss those interactions in more detail, though.